### PR TITLE
Add new item

### PR DESCRIPTION
**Pull request by fashaikh**

### CODE REVIEW

**Comment by msnehagup** [comment]
Why is the runtime version property set to 'na'? Since it's a function app, the runtime version should be 'v1' or 'v2'.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

File: ServerlessLibraryAPI/wwwroot/items.json [hunk 1]
```diff
@@ -587,5 +587,15 @@
...
+  },
+  {
+    "title": "Build a serverless API with Node.js on top of Azure Functions and Azure Cosmos DB",
+    "template": "https://raw.githubusercontent.com/simonaco/nodebinar/master/azuredeploy.json",
+    "repository": "https://github.com/simonaco/nodebinar",
+    "description": "Build a serverless API with Node.js on top of Azure Functions and Azure Cosmos DB",
+    "language": "javascript",
+    "type": "functionapp",
+    "authortype": "Community",
+    "runtimeversion": "na"
```
Comment (msnehagup) [inline ServerlessLibraryAPI/wwwroot/items.json]: please add the appropriate runtime version as well

**Comment by fashaikh** [comment]
> please add the appropriate runtime version as well

Done. Found there were other items that had the version missing. We should automatically read this from the host.json at some point

File: ServerlessLibraryAPI/wwwroot/items.json [hunk 2]
```diff
@@ -526,7 +526,7 @@
     "language": "javascript",
     "type": "functionapp",
     "authortype": "Community",
-    "runtimeversion": "na"
+    "runtimeversion": "v2"
```
Comment (msnehagup) [inline ServerlessLibraryAPI/wwwroot/items.json]: as per the deployment template, this should be v1

File: ServerlessLibraryAPI/wwwroot/items.json [hunk 3]
```diff
@@ -536,7 +536,7 @@
     "language": "csharp",
     "type": "functionapp",
     "authortype": "Community",
-    "runtimeversion": "na"
+    "runtimeversion": "v2"
```
Comment (msnehagup) [inline ServerlessLibraryAPI/wwwroot/items.json]: as per the ARM deployment template, this should be v1

File: ServerlessLibraryAPI/wwwroot/items.json (hunk 2 above)
Comment (fashaikh) [reply ServerlessLibraryAPI/wwwroot/items.json]: I went by the v2.0 mentioned here https://github.com/0GiS0/azure-functions-loves-twilio/blob/master/host.json

File: ServerlessLibraryAPI/wwwroot/items.json (hunk 3 above)
Comment (fashaikh) [reply ServerlessLibraryAPI/wwwroot/items.json]: Same thing here,s ays v2 : https://github.com/0GiS0/audio-azure-functions/blob/master/host.json

**Pull request merged**